jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -59,6 +59,17 @@ with contexts on your new Server 3.0 environment by pushing a fresh commit.
 Once you have successfully run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[reality check],
 notify your team of the new CircleCI UI and URL, if it has changed.
 
+## Frequently Asked Questions
+
+### Where did all my job and build history go?
+* All of your existing jobs and build history have been moved to the Legacy Jobs view.  You can view the complete job history using one of the following methods:
+    ** Selecting Projects -> PROJECT_NAME and selecting the `legacy jobs view` link at the bottom of the project's build history
+    ** Using the following URL pattern: `https://<APP_DOMAIN>/pipelines/github/<ORG>/<PROJECT>/jobs`
```

Comment:
I would not find this or the follow point useful.  But that is just personal preference so I will not block on it.

Reply:
I was trying to put myself in the customer's shoes:
  * As an operator or new 3.0 user, I just finished a migration and start poking around.  I quickly panic because it looks like the migration failed because I don't see any build history at all. 
  * Now that I am poking around, I need to browse my old build information, and getting to it is a little painful.